src/ECER.Engines.Validation/Applications/RenewalValidationConditions/EceAssistant.cs

```diff
@@ -14,6 +14,22 @@ private async Task<List<string>> EceAssistant(Application application)
         break;
 
       case CertificateStatus.ExpiredLessThanFiveYearsAgo:
+        // each application should contain at least one education
+        if (!application.Transcripts.Any())
+        {
+          validationErrors.Add("the application does not have any education");
+        }
```

Comment:
I think we also need to check that there is only 1 education. 

![image](https://github.com/user-attachments/assets/6b61c803-2a6d-4293-8161-ce0088a04be4)